### PR TITLE
Customs conversion for FxaAuth

### PR DESCRIPTION
**Pull request by ameihm0912**
Conversion of existing customs pipeline to operate on FxaAuth data

### CODE REVIEW

File: src/main/java/com/mozilla/secops/parser/EventFilter.java [hunk 1]
```diff
@@ -117,6 +119,10 @@ public Boolean matches(Event e) {
   /**
    * Given any keySelectors return the applicable key from the event
    *
+   * <p>Base64 encoding is applied to key sub elements that are used in the returned key.
```
Comment (ajvb) [inline src/main/java/com/mozilla/secops/parser/EventFilter.java]: Why the base64 encoding?

File: src/main/resources/customs/customsdefault.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "detectors": {
```
Comment (ajvb) [inline src/main/resources/customs/customsdefault.json]: NB: I do wonder if we should make this file private/move it out of the repo.

File: src/main/java/com/mozilla/secops/parser/EventFilter.java (hunk 1 above)
Comment (ameihm0912) [reply src/main/java/com/mozilla/secops/parser/EventFilter.java]: To keep it simple we are just using a string here for the key, and the base64 encoding is used to make sure the input elements do not contain whichever delimiter we are using to separate the elements in it. For example, if a space is used to separate keys we don't want spaces in the elements as well as that will cause problems later splitting them.

File: src/main/resources/customs/customsdefault.json (hunk 2 above)
Comment (ameihm0912) [reply src/main/resources/customs/customsdefault.json]: Possibly at some point, right now this is just the same data you see in https://github.com/mozilla/fxa-customs-server/blob/master/README.md in the policies section. At some point we will likely want to organize this a bit differently.

**Pull request merged**